transposon/data.py

```diff
@@ -41,18 +43,43 @@ def __init__(self, gene_dataframe):
         self.lengths = self.data_frame.Length.to_numpy(copy=False)
         self.chromosomes = self.data_frame.Chromosome.to_numpy(copy=False)
         self.unique_genes = gene_dataframe.index.unique()
-
-    def write(self, filename):
-        """Write to disk."""
-
+        # unique() returns a list, there ought to be one element in the list
+        # and we just want it in string form, so I index on 0 MAGIC NUMBER below
+        self.chrom_of_the_subset = self.data_frame.Chromosome.unique()[0]
+
+    def write(self, filename, key='default'):
+        """Write a Pandaframe to disk.
+
+            filename (str): a string of the filename you want to write.
+            key (str): identifier for the group in the store, since we can
```

Comment:
This should go in the `Args` section. see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html